config.py

```diff
@@ -60,6 +60,7 @@ def init_app(app):
         template_folders = [
             os.path.join(repo_root, 'app', 'templates'),
             os.path.join(repo_root, 'node_modules', 'digitalmarketplace-govuk-frontend', 'govuk-frontend'),
+            os.path.join(repo_root, 'node_modules', 'digitalmarketplace-govuk-frontend', 'digitalmarketplace', 'templates'),
```

Comment:
Can we put this line above `node_modules/digitalmarketplace-govuk-frontend/govuk-frontend`? We should be prioritising templates from the `digitalmarketplace` folder over ones in the latter.